workarounds/skyrimvr.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
The shebang should use /usr/bin/env for better compatibility with non-FHS distros. You can find a little more info by following #416.
```suggestion
#!/usr/bin/env bash
```